flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -90,5 +91,157 @@ class SentryFlutterOptions extends SentryOptions {
         : _cacheDirSize;
   }
 
+  /// Consider disabling [SentryOptions.enableAutoNativeBreadcrumbs] if you
+  /// enable this. Otherwise you might record lifecycle events twice.
+  /// Also consider using [enableBreadcrumbTrackingForCurrentPlatform()]
+  /// instead for more sensible defaults.
+  bool get enableLifecycleBreadcrumbs => _enableLifecycleBreadcrumbs ?? false;
```

Comment:
for all of those options, I'd prefer to set a value by default and not taking any null value, so we'd avoid some NPE checks when passing those values to the Native SDKs